packages/db/test/generator.test.js

```diff
@@ -260,11 +260,12 @@ describe('generator', () => {
     svc.setConfigFields([
       {
         var: 'DATABASE_URL',
-        label: 'What is the connection string?',
-        type: 'string',
-        configValue: 'connectionString'
+        configValue: 'connectionString',
```

Comment:
what is the difference between `configValue` and `var`? Why do we need both?

Reply:
We support both fields as a value that could be an env value (like `DATABASE_URL` in `db`) or a config parameter (like `plugin`)
`configValue` may be always empty, but in future can be useful